system/doc/reference_manual/typespec.xml

```diff
@@ -607,5 +607,29 @@
       of the following form:
     </p>
     <pre>  -spec my_error(term()) -> no_return().</pre>
+
+  <note>
+    <p>Erlang uses the shorthand version <c>_</c> as an anonymous type variable
+    equivalent to <c>term()</c> or <c>any()</c>. For example, the following function
+    <pre>
+  -spec Function(string(), _) -> string().
+    </pre>
+    is equivalent to
+    <pre>
+  -spec Function(string(), any()) -> string().
+    </pre>.
+    However, the identity function defined as
+    <pre>
+  -spec id(_) -> _.
+    </pre>
+    is different from the identity function defined with type variables
+    <pre>
+  -spec id(X) -> X.
+    </pre>
+    where the difference is that the former specifies that the argument and return
+    types can be any type (<c>any()</c>), while the later specified that
+    the argument and return types are of the same type.
```

Comment:
I think this part is unnecessary because there already is a discussion about that kind spec above.